src/lang/en-US.ts

```diff
@@ -18,6 +18,20 @@ const enUSMessages: ResolvedIntlConfig['messages'] = {
     'cta.download': `Download`,
     'cta.login': `Login`,
     'cta.clickHere': `click here`,
+    'cta.moreDetails': `More Details`,
+
+    // Terms
+    'terms.connectors': `Connectors`,
+
+    // Data
+    'data.name': `Name`,
+    'data.description': `Description`,
+    'data.type': `Type`,
+    'data.maintainer': `Maintainer`,
+    'data.updated_at': `Last Updated`,
+
+    // Connectors table
+    'connectors.title': `Table of all connectors`,
```

Comment:
This is used for ARIA tag. Do we want to call out when something is ARIA specific? I flip flopped a lot on that.